index.html

```diff
@@ -44,5 +51,137 @@ <h2>I’am a Software Developer</h2>
       </div>
     </section>
+
+    <section class="my-works">
+      <div class="works-section-a">
+        <h2>My Recent Works</h2>
+
+        <div class="wrapper">
+          <div class="Feature">
+            <img src="src/Img-Placeholder-n1.svg" alt="Feature stories" />
+            <h3>Multi-Post Stories</h3>
+            <p class="Feat-text">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul class="tech-stack-1">
+              <li>CSS</li>
+              <li>HTML</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <p class="btn">See Project</p>
+          </div>
+        </div>
+      </div>
+      <div class="proj-container">
+        <div class="projects">
+          <h3>Profesional Art Printing Data</h3>
+          <p class="Feat-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="tech-stack-2">
+            <li>CSS</li>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <p class="btn">See Project</p>
+        </div>
+        <div class="projects">
+          <h3>Profesional Art Printing Data</h3>
+          <p class="Feat-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="tech-stack-2">
+            <li>CSS</li>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <p class="btn">See Project</p>
+        </div>
+        <div class="projects">
+          <h3>Profesional Art Printing Data</h3>
+          <p class="Feat-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="tech-stack-2">
+            <li>CSS</li>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <p class="btn">See Project</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="about-container"> 
+
+    <section class="about-section">
+      <h2>About Me</h2>
+      <p class="Feat-text font_a">
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <sub-section class="btn about-btn">Get my Resume</sub-section>
```

Comment:
- [ ] Please notice that there is no white border around the design on the Figma template. please kindly remove white border. In this tasks, you should be trying to match the design as closely as possible so keep a keen eye on the details.
## Your Design
![image](https://user-images.githubusercontent.com/18730106/143719894-6e50686d-15b1-4c2d-b512-ae1a8cf57156.png)
## Figma Design
![image](https://user-images.githubusercontent.com/18730106/143719928-b7943ba3-3f1f-4a1e-9df1-1357cd8b2981.png)

- [ ] [OPTIONAL] Please use either an `a` tag or a `button` tag for the "Get my Resume" text. a `sub-section` is not a valid. it's important to try and be semantically correct so please make this change although I've made it optional because the reviewer did net mention it before.